client/UI/FFRKViewInventory.cs

```diff
@@ -304,24 +334,52 @@ private void UpdatePartyGrid(List<DataBuddyInformation> buddies)
 
         void UpdateEquipmentGrid(DataEquipmentInformation[] EquipList)
         {
+            bool needFilter = true;
+            if (comboBoxFilterType.SelectedIndex.Equals(0)) { needFilter = false; }

```

Comment:
I think if I just added it like that, if I selected "Armor", then selected "All", it would still only display "Armor".